owrx/source/afedri.py

```diff
@@ -0,0 +1,87 @@
+from owrx.source.soapy import SoapyConnectorSource, SoapyConnectorDeviceDescription
+from owrx.form.input import Input, TextInput, NumberInput, CheckboxInput
+from owrx.form.input.device import RemoteInput
+from owrx.form.input.converter import OptionalConverter
+from owrx.form.input.validator import RangeValidator
+from typing import List
+
+
+AFEDRI_DEVICE_KEYS = ["rx_mode", "force_set_channel"]
```

Comment:
Thank you for notes. I’ll clear them and come with update. About rx_mode parameter: I was thinking of having it in settings.json but hiding it from UI as this parameter is difficult to explain and it may not be frequently used. Is it acceptable approach to have parameters only in settings.json?

Reply:
no, i'd say that is not a good practice. in general, the files in `/var/lib/openwebrx` should not be edited manually. they are the internal data storage for the web configuration, and any changes should be performed via the web config.

not registering this parameter in the web config probably also means that the web config would delete the parameter when used, which would probably lead to confusion.

for cases like this, the web config has the "optional" inputs, which will not be shown by default, but can be added if necessary from a dropdown. i believe that these keys are already registered as optional, you should have seen how it works, haven't you?

Reply:
Got it. Thanks. Optional keys are controlled by getDeviceOptionalKeys/getProfileOptionalKeys. It is clear.